css/main.css

```diff
@@ -1,3 +1,264 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+.name {
+    font-weight: bolder;
+    text-align: center;
+    margin: 8px 0px;
+}
+
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.navlist {
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+@media (min-width: 768px) {
+    .navlist {
+        display: flex;
+        flex-direction: row;
+        margin-right: -15px;
+        margin-left: -15px;
+    }
+    li {
+        text-align: center;
+    }
+}
+
+.job {
+    text-align: center;
+    margin-top: 0;
+}
+
+header {
+    background-color: antiquewhite;
+    align-items: center;
+}
+
+li {
+    margin-left: 15px;
+    list-style-type: none;
+    text-align: left;
+    font-size: 0.8rem;
+}
+
+.fa-user-circle-o {
+    color: darkcyan;
+    padding-top: 16px;
+    text-align: right;
+}
+
+.login {
+    color: rgb(39, 18, 231);
+    text-align: left;
+    font-weight: bold;
+    font-size: 0.8rem;
+}
+
+.intro {
+    background-image: url("https://unenvironment.s3.us-east-2.amazonaws.com/s3fs-public/inline-images/1.jpg");
+    background-position: center;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-right: -15px;
+    margin-left: -15px;
+}
+
+@media (min-width: 768px) {
+    .intro {
+        min-height:500px ;
+    }
+    .ambition {
+        font-size: small;
+        margin: 4rem 0 0 0;
+    }
+}
+
+.ambition {
+    font-size: small;
+    margin: 4rem 0 0 0;
+}
+
+.succees {
+    font-size: 3rem;
+    font-weight: bolder;
+    margin: 0.5rem 0 0 0;
+}
+
+@media (min-width: 768px) {
+   .succeed {
```

Comment:
I think this should be `succees` so that it works with your HTML. At the moment, this does nothing

Reply:
Yes, actually it should be "success".  I have corrected all of them.